README.md

```diff
@@ -20,10 +21,18 @@ Prerequisites: A GitHub account (for logging into Gitpod)
 * After opening the tool via the notification, the interface loads in a few seconds.
 * Now you're ready to try the trace viewer!
     * Head to the trace viewer tab in the left side menu to get started.
-    * The tool is already loaded with example traces from a set of [Trace Visualisation Labs](https://github.com/tuxology/tracevizlab), so no need to hunt for your own.
+    * The tool is loaded with the example traces from a set of [Trace Visualisation Labs](https://github.com/tuxology/tracevizlab). To analyse your own traces, [download the application](https://github.com/theia-ide/theia-trace-extension#download-the-application) for Linux.
 
 ![gitpod-live-demo-setup](https://raw.githubusercontent.com/theia-ide/theia-trace-extension/master/doc/images/gitpod-live-demo-setup-0.0.2.gif)
 
+## Download an external build of the application 
+If you'd like to explore your own traces, you can **[download a Theia IDE build with this trace viewer extension here](https://www.dorsal.polymtl.ca/files/other/electron-theia-trace-example-app-0.0.1.AppImage)!**
+
+* **Prerequisite: Java 11** (required since this tool reuses the [Eclipse Trace Compass server](https://download.eclipse.org/tracecompass.incubator/trace-server/rcp/?d) which runs on Java)
+   * If you get a confusing error "Error opening serial port ${this.port}. (Port busy)" when you try to run the app, it's likely that Java is missing.  
+* **No compilation or additional downloads necessary!** Just change the AppImage file's permissions to make it executable (command: `chmod +x <filename>`) and run it.
+* **For Linux systems only**
+
 ## Consume the trace viewer extension from npm
 
 We plan to distribute this extension on npm.
```

Comment:
So I guess I can close this pull request ?

Reply:
No, keep this PR for app specific changes. I just fixed items that had been missed in recent PRs. See #503 for my updates. I felt that I should bet the one to fix it since I missed it in previous PRs :-)

Reply:
Done.